### PR TITLE
Foundation Member improvements

### PR DESCRIPTION
**Pull request by ericof**

### CODE REVIEW

**Review by davisagli — COMMENTED** [review]
What's the reason for switching to username as the user id instead of uuid? It seems risky in case there are other references to the old user ids that are not migrated (such as in old content versions, maybe). Also, it's possible to change a github username, so it's not a stable identifier.

**Comment by ericof** [comment]
> What's the reason for switching to username as the user id instead of uuid? 

When we did the migration to Plone 6, instead of using `username` in the user_id factory of `pas.plugins.authomatic`, but foundation members'  local roles were all set using GitHub username.
As it is not possible to map username to UUID, this seems like the best approach (reverse the change)

> It seems risky in case there are other references to the old user ids that are not migrated (such as in old content versions, maybe). Also, it's possible to change a github username, so it's not a stable identifier.

I agree there are risks with this approach, but right now, it seems reasonable. (We could use the GitHub `id` instead of the `username`, but there won't be a mapping for the existing local roles.)

**Review by fredvd — APPROVED** [review]
LGTM.


I think Rikupekka and others have added a few local roles with the uuid to people that wanted to contribute to news items. But we can run a local roles dump script and fix those assignments manually, if still needed.  It can't be more than 10-20. 

Let's merge and see the results first on testing

File: backend/src/ploneorg/src/ploneorg/upgrades/v20230904/__init__.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+            portrait,
+            user.getProperty("login_time", ""),
+            user.getProperty("last_login_time", ""),
+            [g.id for g in api.group.get_groups(user=user)],
+        )
+        users_info[user_info.user_name] = user_info
+    return users_info
+
+
+def content_by_user(users_info: List[UserInfo], site: PloneSite) -> dict:
```
Comment (ericof) [inline backend/src/ploneorg/src/ploneorg/upgrades/v20230904/__init__.py:62]: @fredvd This is where we parse local roles

**Pull request merged**